packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.tsx

```diff
@@ -66,8 +67,17 @@ const ContractTypeWidget = observer(
                     form_source: 'contract_set_up_form',
                     form_name: 'default',
                 });
+                if (!is_dialog_open) setShowDescription(false);
             }
-        }, [is_dialog_open]);
+        }, [is_dialog_open, setShowDescription]);
+
+        React.useEffect(() => {
+            if (show_description) {
```

Comment:
`show_description` is a flag. When user clicks on 'Learn more' button in pop-up, this flag becomes equal to `true`.